set.mm

```diff
@@ -629242,6 +629242,963 @@ standardize vectors to a length (norm) of one, but such definitions require
 $( End $[ set-mbox-sn.mm $] $)
 
 
+$( Begin $[ set-mbox-gg.mm $] $)
+$(
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+  Mathbox for Gino Giotto
+#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
+$)
+
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Minimizing ax-13 usage in a systematic way
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  This section contains proofs of existing theorems in main without requiring
+  ~ ax-13 . Similarly to ~ ax13w , the theorems proved here are weakened by
+  introducing one or more additional dv conditions, hence labeled as xxxw
+  (the pattern xxxv creates naming conflicts with main theorems, despite having
+  different statements. If xxxv is preferred, then the versions in main should
+  be renamed as xxxvv since they all have more dv conditions). These weaker
+  versions can replace the proof steps occupied by their original counterparts
+  with the command "MINIMIZE_WITH xxxw /MAY_GROW". Since these theorems
+  follow the already present proof chain, automatic minimizations should not
+  cause significant proof changes or lengthenings, hence OLD proofs are not
+  necessary (and they would also be unpractical).
```

Comment:
The lack of OLD/ALT proofs caused some people to frawn in the past, so I'm opening this thread to discuss about it. I can change/delete this text if you don't agree with it, but you should be aware that it would discourage me from getting further (unless I have some kind of automation tool, which would be possible in this case). We are talking about thousands changed proofs here, which would correspond to thousands of added OLD/ALT proofs. Please consider that it's not a trivial request to ask (also it would be for relatively minor changes). You can check commit https://github.com/metamath/set.mm/commit/6fc6153e0e4df76bf73bfeef76151d5354bc972c to get an idea of the size of this thing.

Reply:
I'm not sure whether you need this passage at all.  Our conventions page states
`When a proof is shortened automatically (...), then it is not necessary to keep the old proof, nor to add credit for the shortening.`
As for adding lemmas to eliminate ax-13 usage:  Fine from my point of view.  It is our goal to reduce axiom usage - even it comes at the cost of a complete sort of theory developed for that purpose.

Reply:
>  `When a proof is shortened automatically (...), then it is not necessary to keep the old proof, nor to add credit for the shortening.`

This is specific for proof shortenings, but here the new proofs won't be shorter. I think I'll leave this passage unless the conventions will be extended for such cases.